cmake/FreecivInstall.cmake

```diff
@@ -195,6 +195,14 @@ if(UNIX AND NOT APPLE)
       DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/applications
       COMPONENT freeciv21
     )
+
+    # Install database.lua file with the server for FCDB support in a user-writable location
```

Comment:
Won't work very well with packages (only the user making the package will get the file)... I suggest `CMAKE_INSTALL_SYSCONFDIR` which will be under `/etc`.